### ISSUE TITLE
After selecting a version, it is not possible to proceed with the screen reader : 

### ISSUE THREAD

**Issue by optimatrix0** [ReVanced/revanced-manager#2078]
### Bug description

The application works well until you reach the version selection of an application after clicking on one. The add button cannot be clicked with the screen reader active

### Version of ReVanced Manager and version & name of app you are patching

Manager jetpack com pose action 537

### Installation method

None

### ReVanced Manager logs

```shell
N/a
```

### Patch logs

```shell
N/a
```

### Acknowledgements

- [X] This issue is not a duplicate of an existing bug report.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.
- [X] The bug is only related to ReVanced Manager

**Comment by optimatrix0** [comment]
I don't know how to add the tag for the jetpack version with pose so I hope you take my bug seriously because it's true.
